lib/location/location.c

```diff
@@ -161,9 +209,34 @@ void location_config_defaults_set(
 	}
 
 	memset(config, 0, sizeof(struct location_config));
-	config->methods_count = methods_count;
-	config->timeout = 300 * MSEC_PER_SEC; /* 5 minutes */
+	config->interval = CONFIG_LOCATION_REQUEST_DEFAULT_INTERVAL;
+	config->timeout = CONFIG_LOCATION_REQUEST_DEFAULT_TIMEOUT;
 	config->mode = LOCATION_REQ_MODE_FALLBACK;
+
+	/* Handle Kconfig's for method priorities */
+	if (method_types == NULL) {
+		methods_count = 0;
+		method_type = location_str_to_method(CONFIG_LOCATION_REQUEST_DEFAULT_METHOD_FIRST);
+		if (method_type != LOCATION_METHOD_UNKNOWN) {
```

Comment:
This can be also `LOCATION_METHOD_NONE` now, probably that should not be added to the list either.

Reply:
This is not valid with choice symbol implementation.